src/ExerciseOOP/Library.java

```diff
@@ -1,4 +1,77 @@
 package ExerciseOOP;
 
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Scanner;
+
 public class Library {
+
+    public static void main(String[] args) {
+        List<Book> listBook = new ArrayList<>();
+        Book book1 = new Book("Doanh Nghiep The Ki 21", "Robert", "1/1/2000", "Available");
+        Book book2 = new Book("NDCD", "Bob Burg", "1/1/2000", "Available");
+        Book book3 = new Book("NLDKCD", "Robin Sharma", "1/1/2000", "Available");
+        listBook.add(book1);
+        listBook.add(book2);
+        listBook.add(book3);
```

Comment:
I think you should move these code into a function such as `initBooks`

Reply:
I got it, thanks